charts/lws/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: lws
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "v0.4.2"
```

Comment:
is this the only place that we need to update every time we make a release?

Reply:
I think this refers to the lws version, but we need to update the release version as well.

Reply:
In each release, we should update at least two fields: `appVersion` and `version` . We typically update `appVersion` when we're releasing a new version of LWS. But, always update the `version` field with any change to the Helm chart. If there are no other changes to the Helm chart-liske adding more parameters or configuration-then you'll only need to focus on those two.

Reply:
please add release instructions, and I will make them part of the release template when this gets merged